ethexe/service/src/tests.rs

```diff
@@ -1410,7 +1410,7 @@ mod utils {
                 block_time: config.block_time,
             };
             let mut observer =
-                ObserverService::new(&eth_cfg, u32::MAX, db.clone(), Some(blob_reader.clone()))
+                ObserverService::new(&eth_cfg, u32::MAX, db.clone(), blob_reader.clone_box())
```

Comment:
actually I don't see any reason to have it as separated fn, you call it literally 2 times across whole code

Reply:
idk why either. The main point is to get rid of Arc's

Reply:
@grishasobol

Reply:
We never use ObserverService without providing this argument